tests/test_manifest.py

```diff
@@ -1896,7 +1896,79 @@ def test_import_flags_ignore(tmpdir):
     ''', import_flags=ImportFlag.IGNORE)
     assert m.get_projects(['foo'])
 
-def test_import_map_name_whitelist(manifest_repo):
+def test_import_map_name_includelist(manifest_repo):
+    # This tests an example from the documentation which uses
+    # name-includelist.
```

Comment:
There is no such documentation :smiley: 

The previous version of this comment is referring to this example: https://docs.zephyrproject.org/latest/guides/west/manifest.html#example-3-1-downstream-with-name-whitelist

Can you please update the comment until we have a chance to update the documentation to reflect this change? We don't have any such documentation merged yet.

Reply:
Will do.